libs/core-ui/src/lib/util/JointDatasetUtils.ts

```diff
@@ -54,3 +56,7 @@ export interface IJointMeta {
   category: ColumnCategories;
   index?: number;
 }
+
+export interface IDatasetMeta {
+  featureMetaData?: IFeatureMetaData;
```

Comment:
to confirm, will we have more metadata in future other than featureMetaData??

Reply:
yes. likely we will have more such metadata that can come in. For example the feature ranges etc, could also be added here.